rs/ic_os/config/src/lib.rs

```diff
@@ -161,10 +151,7 @@ mod tests {
             "node_reward_type": "type3.1",
             "mgmt_mac": "EC:2A:72:31:A2:0C",
             "deployment_environment": "Mainnet",
-            "logging": {
```

Comment:
As this is still in the type, should we keep some configured examples in these versioned tests just in case? Maybe we make dummy hosts so they don't grep.

Reply:
I've added a configured example back to HOSTOS_CONFIG_JSON_V1_0_0: 7217245342c50b72148fa6fce0ed863814808b0c